### ISSUE TITLE
[Question] JSON file save

### ISSUE THREAD

**Issue by moanhawks** [GrowtopiaNoobs/GrowtopiaServer#780]
is that json file will crash server if me use too more? like gem,level,exp system

**Comment by herothedev** [comment]
No, there is few crash bugs that will crash and you cant fix them by switching to sqlite or BSON.

**Comment by ilyn1337** [comment]
bson is still json but with binary format...

**Comment by playingo** [comment]
if u want to completely avoid CON, PRN crash i would recommend to always use a prefix in those saving files like if u have a world called 'TEST', it will save and load as "__TEST" or something like that. i really dont like windows when its about its filesystem. anyways, hosting a server on linux would fix those issues but then would bring compability issues back so i just recommend using prefixes <3.

**Comment by playingo** [comment]
Oh also, there are tons of other ways on how to crash a server and then dont have to neccessarily do anything with files at all.
